news/11953.feature.rst

```diff
@@ -0,0 +1 @@
+Migrate message_about_scripts_not_on_PATH to use pathlib
```

Comment:
This should mention the slight behavioural change mentioned in the other PR.

Reply:
ok, I change this now